src/components/IstioWizards/IstioWizardDropdown.tsx

```diff
@@ -172,6 +179,7 @@ class IstioWizardDropdown extends React.Component<Props, State> {
         <DropdownButton id="service_actions" title="Actions" onSelect={this.onAction} pullRight={true}>
           {this.canCreate() && this.renderMenuItem(WIZARD_WEIGHTED_ROUTING)}
           {this.canCreate() && this.renderMenuItem(WIZARD_MATCHING_ROUTING)}
+          {this.canCreate() && this.renderMenuItem(WIZARD_SUSPEND_TRAFFIC)}
```

Comment:
nitpicking. What about running those three in a loop?
```typescript
WIZARDS.map((wizard) => this.canCreate() && this.renderMenuItem(wizard)
```

Reply:
On this case I prefer explictly name the options I guess, just to not hide it in a loop.